Snowflake.Data.Tests/UnitTests/QueryContextCacheTest.cs

```diff
@@ -0,0 +1,270 @@
+﻿/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    using Newtonsoft.Json;
+    using NUnit.Framework;
+    using Snowflake.Data.Core;
+    using System;
+    using System.Collections.Generic;
+
+    [TestFixture]
+    class QueryContextCacheTest
+    {
+        private QueryContextCache qcc = null;
```

Comment:
private members should be named with "_"

Reply:
done